apps/i18n/common/en_us.json

```diff
@@ -80,9 +80,9 @@
   "animationSearchPlaceholder": "Search for images...",
   "announcements": "Announcements",
   "announcementHeadingBackToSchool": "Get set up for the new school year",
-  "announcementHeadingBackToSchool2": "Teaching in Virtual and Socially-Distanced Classrooms?",
+  "announcementHeadingBackToSchoolRemote": "Teaching in Virtual and Socially-Distanced Classrooms?",
   "announcementDescriptionBackToSchool": "View our frequently asked questions on how to manage your classroom(s) and start teaching.",
-  "announcementDescriptionBackToSchool2": "View our frequently asked questions on how to start teaching and manage your virtual, hybrid, or in-person classroom(s).",
+  "announcementDescriptionBackToSchoolRemote": "View our frequently asked questions on how to start teaching and manage your virtual, hybrid, or in-person classroom(s).",
```

Comment:
Should it be "manage" or "managing" (to match "teaching")?

Reply:
I think it could go either way - "View questions on how to (start teaching) and (manage your classrooms)", so the questions are about how to start teaching and how to manage your classrooms, but it could also be "View questions on how to start (teaching and managing) your classrooms" - so you are starting to teach your classrooms and starting to manage your classrooms.

Reply:
but now that I've been staring at it for 10 minutes, I think your suggestion sounds better!

Reply:
actually, I can't decide...I'll check with Eric :)

Reply:
went with "managing"!